plugins/woocommerce/src/Blocks/AssetsController.php

```diff
@@ -62,9 +62,14 @@ public function register_assets() {
 		// The price package is shared externally so has no blocks prefix.
 		$this->api->register_script( 'wc-price-format', 'assets/client/blocks/price-format.js', array(), false );
 
-		$this->api->register_script( 'wc-blocks-vendors-frontend', $this->api->get_block_asset_build_path( 'wc-blocks-vendors-frontend' ), array(), false );
-		$this->api->register_script( 'wc-blocks-checkout', 'assets/client/blocks/blocks-checkout.js', array( 'wc-blocks-vendors-frontend' ) );
-		$this->api->register_script( 'wc-blocks-components', 'assets/client/blocks/blocks-components.js', array( 'wc-blocks-vendors-frontend' ) );
+		// Vendor scripts for blocks frontends (not including cart and checkout).
+		$this->api->register_script( 'wc-blocks-frontend-vendors', $this->api->get_block_asset_build_path( 'wc-blocks-frontend-vendors-frontend' ), array(), false );
```

Comment:
Note that because we suffix files in frontend bundles with `frontend` the name of the asset in Webpack's terms is `wc-blocks-frontend-vendors`, and thats what we append to the asset.php file, but the actual filename is suffixed with `frontend`. I did not want to change anything drastic about how we name or generate files because of i18n and caching, but I have renamed the vendor handle in Webpack so that the file is not named `wc-blocks-vendors-frontend-frontend` which seemed... worse 😅 

I could not call it `wc-blocks-vendors` because that handle is already taken by another bundle that does **not** suffix the filename. If we had some foresight in the past maybe we would not have suffixed at all, but named the vendors more uniquely.

Reply:
It's fine to change names, translation will only be impacted locally and those scripts don't have much strings in them.